model/jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -23,6 +23,7 @@
     <jta-data-source>java:jboss/datasources/UnifiedPushDS</jta-data-source>
 
     <mapping-file>META-INF/orm.xml</mapping-file>
+    <shared-cache-mode>ALL</shared-cache-mode>
```

Comment:
Currently we cache all the entities. It would be better to selectively cache entities that worth being cached. That would require further analysis